housekeeping/src/main/java/com/hotels/housekeeping/model/LegacyReplicaPath.java

```diff
@@ -17,22 +17,30 @@
 
 public interface LegacyReplicaPath {
 
-  public long getId();
```

Comment:
Just to understand, why we have removed public now ?

Reply:
It's not necessary in an interface every method by default is public as it is an Interface